Aptoide/aptoide-client-v8
# [APP-1959] E-Skills Revamp

**What does this PR do?**

   Revamps E-Skills image inside vanilla, rebranding and changing existing views and adding new ones.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java

**How should this be manually tested?**

  Test navigation between eskills home bundle, eskills info page and eskills appview.
  Test wallet install as a companion app (install eskills app without having previous wallet installation -> wallet install should be triggered).
  Test general eskills info page layout (see design: https://www.figma.com/file/Ih2iw2LJOlyX931OJWOOPX/Vanilla-V9?type=design&node-id=0-1&mode=dev)

**What are the relevant tickets?**

  Tickets related to this pull-request: 
  - Epic: [APP-1959](https://aptoide.atlassian.net/browse/APP-1959)
  - AppView: [APP-1989](https://aptoide.atlassian.net/browse/APP-1989)
  - Home Bundle: [APP-2035](https://aptoide.atlassian.net/browse/APP-2035)
  - E-Skills Info Page: [APP-2042](https://aptoide.atlassian.net/browse/APP-2042)
  

**Questions:**
   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**


[APP-1959]: https://aptoide.atlassian.net/browse/APP-1959?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-1989]: https://aptoide.atlassian.net/browse/APP-1989?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2035]: https://aptoide.atlassian.net/browse/APP-2035?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2042]: https://aptoide.atlassian.net/browse/APP-2042?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
